app-modules/consultants/src/Policies/DocumentPolicy.php

```
@@ -0,0 +1,42 @@
+
+    public function update(User $user, Document $document): bool
+    {
+        return true;
+    }
+
+    public function delete(User $user, Document $document): bool
+    {
+        return true;
+    }
+
+    public function restore(User $user, Document $document): bool {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)